conda/install.py

```diff
@@ -922,6 +927,8 @@ def move_path_to_trash(path):
         except OSError as e:
             log.debug("Could not move %s to %s (%s)" % (path, trash_dir, e))
         else:
+            if path in linked_data_:
+                del linked_data_[path]
```

Comment:
Ah ok, I see. I will make it handle removing individual packages too then, thanks for the catch.